packages/flutter_test/lib/src/binding.dart

```diff
@@ -1226,24 +1226,52 @@ class AutomatedTestWidgetsFlutterBinding extends TestWidgetsFlutterBinding {
 /// frames.
 ///
 /// These values are set on the binding's
-/// [LiveTestWidgetsFlutterBinding.framePolicy] property. The default is
-/// [fadePointers].
+/// [LiveTestWidgetsFlutterBinding.framePolicy] property.
+///
+/// {@template flutter.flutter_test.frame_policy}
+/// The default is [fadePointers]. Setting this to anything other than
+/// [onlyPumps] means pumping extra frames, which might involve calling builders
+/// more, or calling paint callbacks more, etc, which might interfere with the
+/// test. If you know your test file wouldn't be affected by this, you can set
```

Comment:
If you know your test file wouldn't be => If you know that your test won't be.....

Reply:
Done